SSA/Projects/InstCombine/TacticAuto.lean

```diff
@@ -59,6 +60,10 @@ macro_rules
 macro "simp_alive_undef" : tactic =>
   `(tactic|
       (
+        -- Since I introduced the NSW and NUW flags to add, I need to remove them in the case where they are both
+        -- the default value of false.
+        -- I don't know why Lean isn't smart enough to see that False ∧ x = False
+        try simp only [LLVM.add_reduce]
```

Comment:
Could you make an [MWE](https://leanprover-community.github.io/mwe.html) that describes the problem you ran into? [`false_and`](https://leanprover-community.github.io/mathlib4_docs/Init/SimpLemmas.html#false_and) exists as a simp-lemma, so Lean should definitely do the simplification you describe.